pkg/kubelet/util/swap/swap_util.go

```diff
@@ -46,6 +47,13 @@ func IsTmpfsNoswapOptionSupported(mounter mount.Interface, mountPath string) boo
 			return false
 		}
 
+		if inuserns.RunningInUserNS() {
+			// Turning off swap in unprivileged tmpfs mounts unsupported
+			// https://github.com/torvalds/linux/blob/v6.8/mm/shmem.c#L4004-L4011
```

Comment:
For my education, do you know why this is the case? It's pretty surprising to me that tmpfs mounts can be done under a user namespace, but not with the noswap option.

Reply:
Found this (thanks @kwilczynski for the pointer):
https://www.uwsg.indiana.edu/hypermail/linux/kernel/2304.2/02176.html